### ISSUE TITLE
Blinds to not close completely

### ISSUE THREAD

**Issue by Rainson12** [alxlaxv/SomPi#1]
Hey,
first of all, thanks for your work. It basically seems to work fine. However when closing the blinds, they move down but do not close completely instead they will keep open for a bid. Any idea what could be the reason?

**Comment by Rainson12** [comment]
found it:
accourding to the rts protocoll:
https://pushstack.wordpress.com/somfy-rts-protocol/

0x01 is the "my" key and not the "close":

Value | Button(s) | Description

0x1 | My | Stop or move to favourite position
0x2 | Up | Move up
0x3 | My + Up | Set upper motor limit in initial programming mode
0x4 | Down | Move down
0x5 | My + Down | Set lower motor limit in initial programming mode
0x6 | Up + Down | Change motor limit and initial programming mode
0x8 | Prog | Used for (de-)registering remotes, see below
0x9 | Sun + Flag | Enable sun and wind detector (SUN and FLAG symbol on the Telis Soliris RC)
0xA | Flag | Disable sun detector (FLAG symbol on the Telis Soliris RC)

**Comment by the-lazy-fox** [comment]
Hello @Rainson12,
You mean the following should change?
```
btnDown = 0x1
btnUp = 0x2
btnStop = 0x4
btnProg = 0x8
```

to

```
btnDown = 0x4
btnUp = 0x2
btnStop = 0x1
btnProg = 0x8
```

Thanks for your feedback

**Comment by Rainson12** [comment]
yep exactly

**Comment by tholu** [comment]
Seems to be fixed in #2. Will try it out as well (need to solder the crystal first).
